x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -0,0 +1,277 @@
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.drillIntoAnomaliesTransactionLabel',
+              {
+                defaultMessage:
+                  'Drill into anomalies, transaction details, errors and metrics for deeper analysis.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.useMachineLearningToLabel',
+              {
+                defaultMessage:
+                  'Use machine learning to automatically detect anomalies.',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+    {
+      id: 'infra',
+      label: i18n.translate(
+        'xpack.observability_onboarding.experimentalOnboardingFlow.euiCheckableCard.monitorMyInfrastructureLabel',
+        { defaultMessage: 'Monitor my infrastructure' }
+      ),
+      description: (
+        <ul>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.builtOnPowerfulElasticsearchLabel',
+              {
+                defaultMessage:
+                  'Built on powerful Elasticsearch, stream in and scale infrastructure metrics from your systems, cloud, network, and other infrastructure sources',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.getLogicalAndPhysicalLabel',
+              {
+                defaultMessage:
+                  'Get logical and physical views of your infrastructure topology.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.breakDownApplicationAndLabel',
+              {
+                defaultMessage:
+                  'Break down application and infrastructure silos for faster root cause detection',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+  ];
+
+  const [selectedId, setSelectedId] = useState<string>();
+  const [hoveredId, setHoveredId] = useState<string>();
+
+  const visibleOption = hoveredId
+    ? options.find((option) => option.id === hoveredId)
+    : selectedId
+    ? options.find((option) => option.id === selectedId)
+    : undefined;
+
+  return (
+    <EuiPanel hasBorder>
+      <TitleWithIcon
+        iconType="indexRollupApp"
+        title={i18n.translate(
+          'xpack.observability_onboarding.experimentalOnboardingFlow.strong.startCollectingYourDataLabel',
+          {
+            defaultMessage:
+              'Start collecting your data by selecting one of the following use cases',
+          }
+        )}
+      />
+      <EuiSpacer size="m" />
+      <EuiFlexGroup css={{ margin: 20 }}>
```

Comment:
Not sure about this inline css - is this how it's normally done? IIRC correctly we can use a constant from somewhere to make it easier for things to line up in the layout, but maybe that's a special case here that justifies it here.

Reply:
I didn't find a theme variable with that value. The margin is half of the size of the icon so that the panel is positioned at its center. I'm not aware of a native way of achieving this layout using EUI but happy to change if you can think of one.

Reply:
The `l` avatar is defined by the `xxl` size of the theme: https://github.com/elastic/eui/blob/main/src/components/avatar/avatar.styles.ts#L76

This means we should be able to use `euiTheme.size.xxl / 2` here.